ci-operator/step-registry/openshift/upgrade/gcp/loki/openshift-upgrade-gcp-loki-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-upgrade-gcp-loki
+  steps:
+    pre:
+    - chain: ipi-gcp-pre
+    - ref: ipi-install-loki
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-gcp-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+    env:
+      TEST_COMMAND: "run-upgrade --to-image release:latest"
```

Comment:
Doesn't this arg need to be `${RELEASE_IMAGE_LATEST}` ? Nothing will expand `release:latest` into a digest

Reply:
But nothing would expand `${RELEASE_IMAGE_LATEST}` in an `env` value either, right?  We need to think through recursive expansion.  I guess scripts could `eval` them or something?

Reply:
`${RELEASE_IMAGE_LATEST}` env var is present (not sure what sets it). Some templates do the eval, but luckily its not necessary here

Reply:
Addressed in 7e2932f2fd4, which bakes the "we will use `${RELEASE_IMAGE_LATEST}` for updates" choice into the script itself.  I guess I'm ok with that, but may make more sense to decouple via an `UPDATE_TARGET_RELEASE_IMAGE` dependency like we do for the install version with `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`.  On the other hand, I'm not clear on what it would mean to have that dependency for non-update flows using this step.  I guess we could default it to `release:initial` too, so they'd still only block on that one image?  But they'd have to remember to clobber both if they wanted to change the default for a non-update test...